### DISCUSSION TITLE
Where do i upload my ckpt file?

### DISCUSSION THREAD

**Discussion by andykaufseo** [victorchall/EveryDream-trainer#13] [Q&A]
For some reason the jupyter notebook can't download the 1.5 model from huggingface (can't enter the token because of some javascript issue) - and i don't know where to upload my ckpt file...

**Comment by jmattthew** [comment]
You can put it anywhere.  Then enter the command `downloaded_model_path = "path/model.ckpt"` replacing path/model.ckpt with the actual path you chose
